README.html

```diff
@@ -0,0 +1,1117 @@
+<!DOCTYPE html><html><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1"><style>body {
```

Comment:
what was the purpose of adding this file?

Reply:
Hmm.... I'm not sure how it got in. I'll remove it.